lib/smart_proxy_openscap/openscap_plugin.rb

```diff
@@ -21,6 +21,7 @@ class Plugin < ::Proxy::Plugin
                      :openscap_send_log_file => File.join(APP_ROOT, 'logs/openscap-send.log'),
                      :contentdir => File.join(APP_ROOT, 'openscap/content'),
                      :reportsdir => File.join(APP_ROOT, 'openscap/reports'),
-                     :failed_dir => File.join(APP_ROOT, 'openscap/failed')
+                     :failed_dir => File.join(APP_ROOT, 'openscap/failed'),
+                     :tailoringdir => File.join(APP_ROOT, 'openscap/tailoring')
```

Comment:
Could you please rename this to `tailoring_dir`? I see there are others without underscore, but Ruby convention is to separate words.